aos/closed_loop/ClosedLoopSimulation.ipynb

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
<p>Up to you, but something like including time for individual tasks, with eg. <code>import time</code>  such as </p><p> </p><pre class="ql-syntax" spellcheck="false">print("Preparing PhoSim....")
t0 = time.time()
argString = closeLoopTask.phosimCmpt.getOpdArgsAndFilesForPhoSim(instName)
ttl_time = time.time() - t0
print(f"PhoSim prepared, took {ttl_time} seconds....")
</pre><p>&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Implemented</p>